admin.php

```diff
@@ -0,0 +1,53 @@
+<?php
+$db = new PDO("mysql:host=192.168.20.20;dbname=Portfolio", 'root', '');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$sql = 'SELECT `title`, `img_url`, `date_added` FROM `projects`;';
+
+$query = $db->prepare($sql);;
+$query->execute();
+$result = $query->fetchAll();
+
```

Comment:
Inconsistent whitespace for ending a PHP tag, here you've got a new line and later you've got no new line at the end of another in the same file